pipelines/zendesk/helpers/api_helpers.py

```diff
@@ -0,0 +1,120 @@
+from dlt.common import logger
+from dlt.common.typing import TDataItem, DictStrAny
+from typing import Iterator, Union, Dict, Any
+from zenpy import Zenpy
+from zenpy.lib.api_objects import Ticket
+
+
+class ZendeskCredentials:
```

Comment:
define credentials as in this doc and a test mentioned therein
https://github.com/dlt-hub/dlt/blob/0.2.0a23/docs/technical/secrets_and_config.md#working-with-alternatives-of-credentials-union-types